Source/AliveLibAO/Abe.cpp

```diff
@@ -6882,11 +6883,284 @@ void Abe::State_61_Respawn_42CD20()
+            gElum_507680->field_144 = pSaveData->field_25E_elum_field_144;
+            gElum_507680->field_B2_lvl_number = pSaveData->field_260_elum_lvl_number;
+            gElum_507680->field_B0_path_number = pSaveData->field_262_elum_path_number;
+            gElum_507680->field_A8_xpos = FP_FromInteger(pSaveData->field_268_elum_xpos);
+            gElum_507680->field_AC_ypos = FP_FromInteger(pSaveData->field_26C_elum_ypos);
+            gElum_507680->field_10_anim.field_4_flags.Set(AnimFlags::eBit5_FlipX, pSaveData->field_272_elum_flag & 1);
+            gElum_507680->field_E6_last_anim_frame = 0;
+            gElum_507680->field_120 = 1;
+            gElum_507680->field_122 = pSaveData->field_276_elum_122;
+            gElum_507680->field_128_brain_idx = pSaveData->field_278_brain_idx;
+            gElum_507680->field_12A_brain_state = pSaveData->field_27A_elum_brain_state;
+            gElum_507680->field_12C_honey_xpos = static_cast<short>(pSaveData->field_27C_honey_xpos);
+            gElum_507680->field_146 = pSaveData->field_280_honey_ypos;
+            gElum_507680->field_130 = pSaveData->field_284_elum_field_130;
+
+            gElum_507680->field_170_flags.Set(Elum::Flags_170::eFoundHoney_Bit4, pSaveData->field_28B & 1);
+            gElum_507680->field_170_flags.Set(Elum::Flags_170::eBit3, pSaveData->field_28A_some_flag & 1);
+            gElum_507680->field_170_flags.Set(Elum::Flags_170::eBit2, pSaveData->field_289 & 1);
+            if (gElum_507680->field_B0_path_number == sActiveHero_507678->field_B0_path_number)
+            {
+                if (pSaveData->field_270 != -1)
+                {
+                    PathLine* pLine = nullptr;
+                    FP hitX = {};
+                    FP hitY = {};
+                    if (sCollisions_DArray_504C6C->RayCast_40C410(
+                        gElum_507680->field_A8_xpos,
+                        gElum_507680->field_AC_ypos - FP_FromInteger(60),
+                        gElum_507680->field_A8_xpos,
+                        gElum_507680->field_AC_ypos + FP_FromInteger(60),
+                        &pLine,
+                        &hitX,
+                        &hitY,
+                        1 << pSaveData->field_270))
+                    {
+                        gElum_507680->field_F4_pLine = pLine;
+                        gElum_507680->field_FC_current_motion = eElumStates::State_1_Idle_412990;
+                        gElum_507680->field_E4_previous_motion = eElumStates::State_1_Idle_412990;
+                    }
+                    else
+                    {
+                        gElum_507680->field_FC_current_motion = eElumStates::State_21_Land_414A20;
+                        gElum_507680->field_E4_previous_motion = eElumStates::State_21_Land_414A20;
+                    }
+                }
+            }
+            else
+            {
+                gElum_507680->field_F4_pLine = (PathLine *) -2;
```

Comment:
reinterpret_cast, this is looking like some OWI hack as well, setting a pointer to -2 is very suspect